### DISCUSSION TITLE
skywalking alerting rules

### DISCUSSION THREAD

**Discussion by tunbb** [apache/skywalking#12421] [Q&A]
There is a rule introduction in https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-alarm/#rules.
Root operation of the expression must be a Compare Operation and a threshold is needed. But how to alarm the metrics that has no values? such as endpoint_cpm, I want to find out the endpoint that is not called last 2 minutes

**Comment by wu-sheng** [comment]
If you want to check a service recent 2 minutes has not traffic within 10 mins window, then you need relative function, such as count, or is_present

But you will find out it is not easy to set it accurately.

This is not a good practice, actually.
